### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by ry5n**
Status: **Ready for Review**
Owner: @ry5n
Reviewers: @jeffkamo @fractaltheory @donnielrt 

Since this is a document, you can [view the rich diff](https://github.com/mobify/stencil-select/pull/7/files?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8)
## Changes
- Add readme. Outline:
  - Title
  - Description (should always say “component for AdpativeJS”)
  - Demo link (not working currently)
  - Requirements
  - Installation
  - Usage
  - API (subsections for Dust params & bodies, Sass variables, UI options & event)
  - Contributing
## Jira Tickets:
- [CSOPS-1249](https://mobify.atlassian.net/browse/CSOPS-1249) 
## Todos:
- [x] +1 @jeffkamo
- [x] +1 @fractaltheory
### Feedback:

_none so far_

### CODE REVIEW

**Comment by jeffkamo** [comment]
This readme looks great! Very thorough and detailed. Love the varied examples and usages, not to mention all aspects of the API.

File: README.md [hunk 1]
```diff
@@ -1,27 +1,130 @@
...
-- Sass
-- AMD loader
-- Dust templating
+- AdaptiveJS 2.1 or greater
 
 ## Installation
 
-Installation instructions. Be as brief as possible without leaving out necessary requirements.
+```shell
+cd my-adaptive-project
```
Comment (jeffkamo) [inline README.md]: Pretty nitpicky, but I feel like slash delimited paths for shell commands like this are a bit clearer. `cd my/adaptive/project`. Not a big deal though.

**Comment by jeffkamo** [comment]
Looks solid. I don't see any issues worth bringing up. +1

File: README.md [hunk 2]
```diff
@@ -1,27 +1,130 @@
...
+
+### UI events
+
+Event name        | Data object keys     | Event description
+:---------------- | :------------------- | :----------------
+select:update     | value (after update) | Emitted when the underlying native select changes.
+
+## Contributing
+
+First, read the Adaptive component documentation, especially the pages on creating components and the Stencil authoring guide. Then, clone the repo:
```
Comment (fractaltheory) [inline README.md:116]: Once we have the Adaptive component documentation live and have URLs to use, we can add a link here.

**Comment by fractaltheory** [comment]
Great work @ry5n! I pointed out one thing above about the component documentation link.

File: README.md (hunk 2 above)
Comment (ry5n) [inline README.md:116]: Totally

Comment (jeffkamo) [inline README.md:116]: yup, agreed

**Comment by jeffkamo** [comment]
@fractaltheory does that count as a +1? :)

**Comment by fractaltheory** [comment]
Yep :+1:

**Comment by ry5n** [comment]
@fractaltheory Thanks. Good catch about adding the docs link.

**Pull request merged**